packages/sdk/src/types/Promotions.ts

```diff
@@ -51,7 +51,7 @@ export interface PromotionsCreate {
 	promotion?: {
 		tiers: {
 			name: string
-			banner: string
+			banner?: string
```

Comment:
Is this field not propagated at all? It's braking change.

Reply:
Its not a breaking change since this is request payload, not response.

Reply:
Will be added to changelog